operations/flags.go

```diff
@@ -274,6 +275,13 @@ func addUncommittedChangesFlag(flags ...cli.Flag) []cli.Flag {
 	})
 }
 
+func addPreserveCommitsFlag(flags ...cli.Flag) []cli.Flag {
+	return append(flags, cli.BoolFlag{
+		Name:  preserveCommitsFlag,
```

Comment:
I couldn't come up with a shorter version that wasn't taken. Can you think of one?
I don't think this will be a common thing (I imagine not many users include multiple commits in commit queue patches, though I haven't investigated it empirically).

Reply:
Hmmm that's true. If only this was the default and then the flag could be --squash :) I'm fine with this.